src/Xeno/Errors.hs

```diff
@@ -34,6 +34,7 @@ bshow :: Show a => a -> BS.ByteString
 bshow = BS.pack . show
 
 {-# INLINE CONLIKE getStartIndex #-}
+-- FIXME remove this; there's no offset in the bytestring.
 getStartIndex :: BS.ByteString -> Int
 getStartIndex (PS _ from _) = from
```

Comment:
This is a pattern synonym in bytestring >= 11 where `from` is always 0. It's fine to leave this as is but would be misleading in error messages.